app/graphql_server/main.go

```diff
@@ -27,13 +28,13 @@ func main() {
 		{Runner: users[0], Game: games[0], Time: 17 * time.Millisecond},
 		{Runner: users[1], Game: games[0], Time: 3 * time.Hour},
 	}
-	server := &(server{
-		games: games,
-		users: users,
-		runs:  runs,
+	server := &(graphql_server.Server{
```

Comment:
Jsyk: the `(` aren't needed here.

Reply:
Neat, I'm not sure why I did this by reflex. The next person to touch this file can fix (or if I need to make fixes I'll remove them)